src/main.rs

```diff
@@ -35,6 +36,7 @@ pub struct CliOpts {
     ///   "governance_canister_id": "rrkah-fqaaa-aaaaa-aaaaq-cai",
     ///   "ledger_canister_id": "ryjl3-tyaaa-aaaaa-aaaba-cai",
     ///   "root_canister_id": "r7inp-6aaaa-aaaaa-aaabq-cai"
+    ///   "dapp_canister_id_list": [],
```

Comment:
There were two suggestions I applied to your last PR before merging it so this may cause some conflicts when trying to merge this one. I would suggest removing these changes and rebasing on master. Feel free to reach out to me for help

Reply:
Will do, thanks